src/Controller/CRUDController.php

```diff
@@ -967,6 +1022,18 @@ public function getRequest()
         return $this->container->get('request');
     }
 
+    /**
+     * Gets a container configuration parameter by its name.
+     *
+     * @param string $name The parameter name
+     *
+     * @return mixed
+     */
+    protected function getParameter($name)
+    {
+        return $this->container->getParameter($name);
+    }
```

Comment:
What is this new method? Something really needed?

Reply:
That method is the only one *still* in the FrameworkBundle `Controller` class, even in Symfony 4.0.

Since all that polyfill stuff might be removed in the next major version, implementing this method explicitely is required to maintain BC.